tests/test_interface.rb

```diff
@@ -1328,8 +1330,6 @@ def interface_test_data
         default_shutdown:    true,
         switchport:          :disabled,
         default_switchport:  :disabled,
-        access_vlan:         DEFAULT_IF_ACCESS_VLAN,
```

Comment:
Why are you removing these?

Reply:
For loopback and vlan interfaces, access_vlan is not an option. For ethernet and port-channel it is available if the switchport mode is turned on.